packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -983,6 +992,7 @@ const SchedulerForm: FC<Props> = ({
                                                         data={slackChannels}
                                                         searchable
                                                         creatable
+                                                        limit={10}
```

Comment:
Hi, i put this because caz i remember it was not very smooth when rendering the dropdown list without this limit set when deal with the large data set, yea but let me confirm that, if its still the case i will expand the limit to a bigger number.

Reply:
yea, i tested with 99999 channels without `limit` the input box is not responding smoothly.

in this clip i was keep clicking the input box and this is what it looks like.

https://github.com/user-attachments/assets/8b3a3cf0-90d8-4135-bf5c-ac30832c1bba